cmd/dashboard.go

```diff
@@ -48,6 +50,18 @@ var clearDashboards = &cobra.Command{
 	},
 }
 
+// askForConfirmation prompts user to confirm operation
+func askForConfirmation(msg string) {
+	fmt.Printf(msg)
+	r := bufio.NewReader(os.Stdin)
+	ans, _ := r.ReadString('\n')
+	response := strings.ToLower(ans)
+	if response[0] != 'y' && response[0] != 'Y' {
+		log.Infof("Goodbye")
+		os.Exit(0)
```

Comment:
Might be better to have this return true/false and let the caller decide what to do based on that. In this particular instance we want to exit but I can imagine a case down the line where you want to confirm something and if they decline maybe you still want to continue doing some other operation.

Reply:
well, that's fair.  I just switched it to have the user pass in a validation fucntion.